### DISCUSSION TITLE
Eliminate prints in google colab

### DISCUSSION THREAD

**Discussion by ShayAdaddi** [sintel-dev/Orion#543] [Q&A] [answered]
Hi,

How can I eliminate all the uninteresting prints like the attached image
![image](https://github.com/sintel-dev/Orion/assets/60758697/75c5a83a-196d-4db1-94f9-5238feffab6c)

I want to see output like in the tutorials (clean with epoch and loss )

**Comment by sarahmish** [comment, marked as answer]
Hi @ShayAdaddi – this seems like a log produced by tensorflow or another dependency. Can you check how to turn off logging / debugging for such package? 

This documentation is showing the same output you are having
https://www.tensorflow.org/api_docs/python/tf/debugging/set_log_device_placement

**Reply by ShayAdaddi** [reply]
Hi it was debug option in tf 
Thanks!
